endpoint-deprecation/README.md

```diff
@@ -0,0 +1,112 @@
+
+To see timeseries graphs for endpoint latency, error and throughput, run the following command and then click the `Live View` link at the top:
+
+```
+px live pxbeta/service_endpoints -- -start_time '-30m' -service 'default'
+```
+
+## Full-body HTTP/2 Requests for the Specified Endpoint
+
+Copy the `service_endpoint_requests` folder locally, then run:
+
+```
+px live -f service_endpoint_requests -- -start_time '-30m' -service 'default' -endpoint '/v1/catalog/*/details' -max_num_records 1000
+```
+
+<img src=".readme_assets/service_endpoint_requests.png" alt="Sample of requests sent to an endpoint.">
+
+To inspect truncated table cells (e.g. `req_headers`), select the cell then press `enter`. To exit the expanded view, use `esc`.
+
+## A List of Unique Request Header Field Values
+
+Copy the `unique_req_header_values` folder locally, then run:
+
+```
+px live -f unique_req_header_values -- -start_time '-30m' -service 'default' -endpoint '/v1/catalog/*/details'
+```
+
+<img src=".readme_assets/unique_req_header_values.png" alt="List of unique request header field values.">
+
+This script only examines two request header fields (Referer and Api-Key), but the `.pxl` file can be easily modified to inspect other fields.
+
+## How to Interact with the Live CLI
+
+For more information on how to interact with the Live CLI, including sorting columns, expanding truncated table cells and more, check out the [reference docs](https://docs.px.dev/using-pixie/using-cli/#use-the-live-cli).
+
+## How to Run Scripts in the Live UI
+
+PxL scripts can be run using the [CLI](https://docs.px.dev/using-pixie/using-cli/#use-the-live-cli) or [Live UI](https://docs.px.dev/using-pixie/using-live-ui). This tutorial demonstrated the CLI. To run these scripts in the Live UI:
+
+- Open the Live UI.
+- Select the `Scratch Pad` script from the script drop-down menu in the top left.
+- Open the script editor using `ctrl+e` (Windows, Linux) or `cmd+e` (Mac).
+- Copy the contents of the `.pxl script` file into the “PxL Script” tab of the script editor.
+- Copy the contents of the `vis.json` file into the “Vis Spec” tab of the script editor.
+- Run the script using the RUN button in the top right or using the `ctrl+enter` (Windows, Linux) or `cmd+enter` (Mac) shortcut.
+- Hide the script editor using `ctrl+e` (Windows, Linux) or `cmd+e` (Mac).
+
+## A Note on Data Retention
+
+Note that the PxL scripts above only examine the last 30 minutes of traffic. This time window can be extended by changing the `start_time` variable (e.g. to use `-3h`). Pixie stores collected data locally on the nodes in your cluster, so data retention is limited to 24 hours or less (depending on the volume of traffic your cluster sees). For more visibility, use [Pixie’s API](https://docs.px.dev/using-pixie/api-quick-start/) to routinely scan for endpoint traffic and log the results (or send a [Slackbot alert](https://docs.px.dev/tutorials/integrations/slackbot-alert/)).
+
```

Comment:
this entire sections should be cut and replaced with reference links

Reply:
This material doesn't currently exist in the docs. I'll work on adding it in a separate PR.